src/appmixer/google/drive/bundle.json

```diff
@@ -46,7 +46,7 @@
         "1.4.5": [
             "MoveFileOrFolder: fixes - prevent 'Increasing the number of parents is not allowed', added DEBUG info message - logging file metadata and request params. Set the DEBUG configuration key in the Backoffice to true to see the messages."
         ],
-        "1.4.6": [
+        "1.4.7": [
```

Comment:
@harshaio  please add a new release note, the 1.4.6 is out already, nevertheless, this is a regular update, we should describe the change. Note that you can check which version is released in the master branch in the private repo: https://github.com/clientIO/appmixer-components/tree/master